hack/update-vendor.sh

```diff
@@ -24,6 +24,10 @@ cd "$(pwd -P)"
 KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
+# Get all the default Go environment.
+kube::golang::setup_env
+kube::golang::verify_go_version
+
 # Turn off workspaces until we are ready for them later
 export GOWORK=off
```

Comment:
do we still need to set this off here?

Reply:
yes, the processing of modules is somewhat different with GOWORK on vs. off (or was when I wrote this)